Directory.Build.props

```
@@ -17,11 +17,11 @@
 
   <Target Name="ResetTargetFrameworks" AfterTargets="FindUnity" BeforeTargets="BeforeResolveReferences">
     <!-- With 2022 Unity packages are already targeting ns2.1 and Unity itself only has that as an option so we must target ns2.1 here too: -->
-    <PropertyGroup Condition="$(UnityVersion.StartsWith('2022'))">
+    <PropertyGroup Condition="$(UnityVersion.StartsWith('2021.2')) or $(UnityVersion.StartsWith('2022'))">
       <TargetFrameworks>netstandard2.1</TargetFrameworks>
     </PropertyGroup>
     <!-- Lowest version supported at this time is 2019 and up to 2021 ns2.0 is the only option. -->
-    <PropertyGroup Condition="$(UnityVersion.StartsWith('2019')) or $(UnityVersion.StartsWith('2020')) or $(UnityVersion.StartsWith('2021'))">
+    <PropertyGroup Condition="$(UnityVersion.StartsWith('2019')) or $(UnityVersion.StartsWith('2020')) or $(UnityVersion.StartsWith('2021.1'))">
```

OR if it doesn't work with 2021.2
```suggestion
    <PropertyGroup Condition="$(UnityVersion.StartsWith('2019')) or $(UnityVersion.StartsWith('2020'))">
```